search-app/src/cmr/search/data/query_to_elastic.clj

```diff
@@ -371,7 +374,12 @@
        ;; We only include this if one of the others is present
        (when (and (or use-temporal-sort? use-keyword-sort?)
                   (sort-use-relevancy-score))
-         [{:usage-relevancy-score {:order :desc :missing 0}}])))))
+         [{:usage-relevancy-score {:order :desc :missing 0}}])
+       ;; If end-date is nil, collection is ongoing so use today so ongoing
+       ;; collections will be at the top
+       (when use-keyword-sort?
+         [{:end-date {:order :desc
+                      :missing (time-coerce/to-long (time/today))}}])))))
```

Comment:
It looks like temporal for collections can have a time component. Shouldn't we use `(time/now)` instead of `(time/today)`?